Library/Basic Set/Basic Set Equipment.eqp

```diff
@@ -5254,6 +5254,24 @@
 				"extended_weight": "0.144 lb"
 			}
 		},
+		{
+			"id": "ab48f4e5-1a6d-404c-b08c-03ad707e4976",
+			"type": "equipment",
+			"description": "Complete Wardrobe (Status 0)",
+			"reference": "B266",
+			"notes": "One to four sets of ordinary clothes, plus nightclothes, one set each of formal wear and winter clothes, and usually\nat least one outfit (lab coat, uniform, gym clothes, etc.) appropriate to your job or hobbies.",
```

Comment:
Would consider removing this line for copyright reasons as it's lifted straight from the book.

Reply:
The line has been changed as requested.